src/app/shared/helpers/item-route.ts

```diff
@@ -56,11 +56,11 @@ export function itemRouteFromParams(params: ParamMap): ItemRoute|ItemRouteError
   const attemptId = params.get(attemptParamName);
   const parentAttemptId = params.get(parentAttemptParamName);
 
-  if (id === null) return { tag: 'error', id: undefined };
+  if (!id) return { tag: 'error', id: undefined }; // null or empty
   if (pathAsString === null) return { tag: 'error', id: id };
   const path = pathAsString === '' ? [] : pathAsString.split(',');
-  if (attemptId !== null) return { id: id, path: path, attemptId: attemptId };
-  if (parentAttemptId !== null) return { id: id, path: path, parentAttemptId: parentAttemptId };
+  if (attemptId) return { id: id, path: path, attemptId: attemptId }; // not null nor empty
+  if (parentAttemptId) return { id: id, path: path, parentAttemptId: parentAttemptId }; /* not null nor empty */
```

Comment:
`// not null nor empty`
` /* not null nor empty */`
Feels weird to not have those both comments commented in the same way